### ISSUE TITLE
[2026-02-04] Daily Papers (54 papers)

### ISSUE THREAD

**Issue by Shininglay** [Shininglay/get-daily-noti#8]
# 📅 Daily Paper Update: 2026-02-04

**今日发现 54 篇近期(30天内)相关论文**

---

### [Sensing Spin Precession with Free Electrons](https://pubs.acs.org/doi/10.1021/acsnano.5c13351?af=R)
- **关键词**: `sensing`
- **发布时间**: 2026-01-20T08:00:00Z

### [Van der Waals Interactions between Nonpolar Alkyl Chains and Polar ZnO Surfaces in Gas Sensing Dynamics of Aliphatic Carboxylic Acids](https://pubs.acs.org/doi/10.1021/acsnano.5c15490?af=R)
- **关键词**: `sensing`
- **发布时间**: 2026-01-16T08:00:00Z

### [Mechanically Strengthened Graphene Oxide: Covalent Organic Framework Membranes for Monovalent/Divalent Cation Selectivity via Electrodialysis](https://pubs.acs.org/doi/10.1021/acsnano.5c17289?af=R)
- **关键词**: `GaN`
- **发布时间**: 2026-01-21T08:00:00Z

### [Stimulus-Responsive NOT Gate for Single-Rail DNA Logic Circuits and Biosensing](https://pubs.acs.org/doi/10.1021/acsnano.5c17487?af=R)
- **关键词**: `sensing`
- **发布时间**: 2026-01-23T08:00:00Z

### [Interfacing and Growth Pathways of Metal–Organic Frameworks with Morphological Evolution](https://pubs.acs.org/doi/10.1021/acsnano.5c17654?af=R)
- **关键词**: `GaN`
- **发布时间**: 2026-01-17T08:00:00Z

### [Principles of Use of Tensile J-Curve Materials in Antagonistic Arrangements](https://arxiv.org/abs/2602.00260)
- **关键词**: `sensing`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Ultrasensitive, universal single-ion nanodetector](https://arxiv.org/abs/2602.00406)
- **关键词**: `sensor`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Enhancing Imaging Depth and Sensitivity in Reflectance Mode Near Infrared Optical Imaging with Scatter Reducing Agents](https://arxiv.org/abs/2602.00368)
- **关键词**: `sensing`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Fidelity and quantum geometry approach to Dirac exceptional points in diamond nitrogen-vacancy centers](https://arxiv.org/abs/2602.00666)
- **关键词**: `sensing`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [An Open-Source Framework for Measurement and Analysis of Nanoscale Ionic Transport](https://arxiv.org/abs/2602.00806)
- **关键词**: `GaN`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Steady-State Exceptional Point Degeneracy and Sensitivity of Nonlinear Saturable Coupled Oscillators](https://arxiv.org/abs/2601.13638)
- **关键词**: `sensor`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Quantitative mobile gamma-ray spectrometry through Bayesian inference](https://arxiv.org/abs/2512.18769)
- **关键词**: `sensing`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Experimental Validation of SBFD ISAC in an FR3 Distributed SIMO Testbed](https://arxiv.org/abs/2602.00054)
- **关键词**: `sensing`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [SurfelSoup: Learned Point Cloud Geometry Compression With a Probablistic SurfelTree Representation](https://arxiv.org/abs/2602.00186)
- **关键词**: `GaN`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Advanced Geometric Correction Algorithms for 3D Medical Reconstruction: Comparison of Computed Tomography and Macroscopic Imaging](https://arxiv.org/abs/2602.00220)
- **关键词**: `GaN`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Benchmarking Vanilla GAN, DCGAN, and WGAN Architectures for MRI Reconstruction: A Quantitative Analysis](https://arxiv.org/abs/2602.00221)
- **关键词**: `GaN`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [CMANet: Channel-Masked Attention Network for Cooperative Multi-Base-Station 3D Positioning](https://arxiv.org/abs/2602.00696)
- **关键词**: `sensing`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Cognitive-Flexible Control via Latent Model Reorganization with Predictive Safety Guarantees](https://arxiv.org/abs/2602.00812)
- **关键词**: `sensing`, `GaN`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [mmWave Sensing for Detecting Movement Through Thermoplastic Masks During Radiation Therapy Treatment](https://arxiv.org/abs/2602.00917)
- **关键词**: `sensing`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Coordinate-conditioned Deconvolution for Scalable Spatially Varying High-Throughput Imaging](https://arxiv.org/abs/2602.01065)
- **关键词**: `sensor`, `GaN`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Digital and Hybrid Precoding and RF Chain Selection Designs for Energy Efficient Multi-User MIMO-OFDM ISAC Systems](https://arxiv.org/abs/2602.01121)
- **关键词**: `sensing`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Mismatch Analysis and Cooperative Calibration of Array Beam Patterns for ISAC Systems](https://arxiv.org/abs/2602.01293)
- **关键词**: `sensing`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [LMI Optimization Based Multirate Steady-State Kalman Filter Design](https://arxiv.org/abs/2602.01537)
- **关键词**: `sensor`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Hyperspectral Image Fusion with Spectral-Band and Fusion-Scale Agnosticism](https://arxiv.org/abs/2602.01681)
- **关键词**: `sensor`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Obstacle Detection at Level Crossings under Adverse Weather Conditions -- A Survey](https://arxiv.org/abs/2602.01974)
- **关键词**: `sensor`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Neurophysiological effects of museum modalities on emotional engagement with real artworks](https://arxiv.org/abs/2602.02086)
- **关键词**: `sensing`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [RIS-Aided Wireless Amodal Sensing for Single-View 3D Reconstruction](https://arxiv.org/abs/2602.02148)
- **关键词**: `sensing`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [A Novel ISAC Waveform Based on Orthogonal Delay-Doppler Division Multiplexing with FMCW](https://arxiv.org/abs/2602.02248)
- **关键词**: `sensing`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Representation Learning Enhanced Deep Reinforcement Learning for Optimal Operation of Hydrogen-based Multi-Energy Systems](https://arxiv.org/abs/2602.00027)
- **关键词**: `GaN`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Real-Time Human Activity Recognition on Edge Microcontrollers: Dynamic Hierarchical Inference with Multi-Spectral Sensor Fusion](https://arxiv.org/abs/2602.00152)
- **关键词**: `sensor`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Enhancing Imaging Depth and Sensitivity in Reflectance Mode Near Infrared Optical Imaging with Scatter Reducing Agents](https://arxiv.org/abs/2602.00368)
- **关键词**: `sensing`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [TLDiffGAN: A Latent Diffusion-GAN Framework with Temporal Information Fusion for Anomalous Sound Detection](https://arxiv.org/abs/2602.01060)
- **关键词**: `GaN`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [AdaptNC: Adaptive Nonconformity Scores for Uncertainty-Aware Autonomous Systems in Dynamic Environments](https://arxiv.org/abs/2602.01629)
- **关键词**: `sensor`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Time2Vec-Integrated Transformer for Robust Gesture Recognition from Low-Density sEMG](https://arxiv.org/abs/2602.01855)
- **关键词**: `sensor`, `sensing`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Online Fine-Tuning of Pretrained Controllers for Autonomous Driving via Real-Time Recurrent RL](https://arxiv.org/abs/2602.02236)
- **关键词**: `sensor`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Bridging the Sim-to-Real Gap with multipanda ros2: A Real-Time ROS2 Framework for Multimanual Systems](https://arxiv.org/abs/2602.02269)
- **关键词**: `soft robotics`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Collision-free Source Seeking and Flocking Control of Multi-agents with Connectivity Preservation](https://arxiv.org/abs/2301.04576)
- **关键词**: `sensor`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Future frame prediction in chest and liver cine MRI using the PCA respiratory motion model: comparing transformers and dynamically trained recurrent neural networks](https://arxiv.org/abs/2410.05882)
- **关键词**: `neural interface`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [System Identification for Virtual Sensor-Based Model Predictive Control: Application to a 2-DoF Direct-Drive Robotic Arm](https://arxiv.org/abs/2505.23138)
- **关键词**: `sensor`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [AI-Driven Low-Altitude Economy: Spectrum, Mobility, and Validation](https://arxiv.org/abs/2506.01378)
- **关键词**: `sensing`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Bio-inspired density control of multi-agent swarms via leader-follower plasticity](https://arxiv.org/abs/2507.15781)
- **关键词**: `GaN`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [RDDM: Practicing RAW Domain Diffusion Model for Real-world Image Restoration](https://arxiv.org/abs/2508.19154)
- **关键词**: `sensor`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Distributed Koopman Operator Learning from Sequential Observations](https://arxiv.org/abs/2509.20071)
- **关键词**: `sensing`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [AI-Based Stroke Rehabilitation Domiciliary Assessment System with ST_GCN Attention](https://arxiv.org/abs/2510.00049)
- **关键词**: `sensor`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [SatFusion: A Unified Framework for Enhancing Remote Sensing Image via Multi-Frame and Multi-Source Image Fusion](https://arxiv.org/abs/2510.07905)
- **关键词**: `sensing`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Trustworthy Equipment Monitoring via Cascaded Anomaly Detection and Thermal Localization](https://arxiv.org/abs/2512.24755)
- **关键词**: `sensor`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Probability-Aware Parking Selection](https://arxiv.org/abs/2601.00521)
- **关键词**: `sensing`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Hybrid Beamforming Optimization for MIMO ISAC based on Prior Distribution Information](https://arxiv.org/abs/2506.07869)
- **关键词**: `sensing`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [An Extended VIIRS-like Artificial Nighttime Light Data Reconstruction (1986-2024)](https://arxiv.org/abs/2508.00590)
- **关键词**: `sensor`, `sensing`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Trade-offs between structural richness and communication efficiency in music network representations](https://arxiv.org/abs/2509.14053)
- **关键词**: `GaN`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Estimating Respiratory Effort from Nocturnal Breathing Sounds for Obstructive Sleep Apnoea Screening](https://arxiv.org/abs/2509.14944)
- **关键词**: `sensor`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [BOLT-GAN: Bayes-Error-Motivated Objective for Stable GAN Training](https://arxiv.org/abs/2510.25609)
- **关键词**: `GaN`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [RF-MatID: Dataset and Benchmark for Radio Frequency Material Identification](https://arxiv.org/abs/2601.20377)
- **关键词**: `sensor`, `optical sensor`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500

### [Evaluating Spatialized Auditory Cues for Rapid Attention Capture in XR](https://arxiv.org/abs/2601.21264)
- **关键词**: `sensor`
- **发布时间**: Tue, 03 Feb 2026 00:00:00 -0500
